doc/ANN.md

```diff
@@ -118,11 +118,13 @@ Wallet
 Cpuminer
 --------
 Bech32 address is by default and strongly recommended. `-t1` uses 1 thread. If you want more hash, increase this number.
-- Solo mining: Make a file `sugarchain.conf`, restart your wallet and run cpuminer-opt-sugarchain (RPC=`34229`, testnet5 RPC=`44229`, regtest RPC=`45339`)
+- Solo mining: Open wallet and go to Settings>Options>Wallet Configuration and enter the code below , restart your wallet and run cpuminer-opt-sugarchain (RPC=`34229`, testnet5 RPC=`44229`, regtest RPC=`45339`)
 ```
 server=1
 rpcuser=username
 rpcpassword=password
+rpcallowip=127.0.0.1
```

Comment:
i think rpcallowip is not needed? could you check it again?

Reply:
I will check when i get home and test whether miner works without allowip